servo-tidy.toml

```diff
@@ -61,7 +61,6 @@ packages = [
     "env_logger",
 
     # Stylo uses phf 0.11, while script and other dependencies use 0.10.
```

Comment:
Can this comment be removed now?

Reply:
There is still string_cache that depends on 0.10, do I simply remove the reference to script?

Reply:
Yes I think we should at least remove the reference to script, since this PR will make it untrue otherwise. If you are sure there is only one dependency left, we might also as well name it here.

Reply:
Done, checked with cargo tree.